docs/GettingStartedDocs/Contributors/WindowsManualInstallPrereqs.md

```diff
@@ -0,0 +1,83 @@
+
+# Manually Installing Open Enclave Prerequisites for Windows on a System which supports SGX
+
+## Platform requirements
+- A system with support for SGX1 or SGX1 with Flexible Launch Control (FLC).
+ Note: To check if your system has support for SGX1 with or without FLC, please look [here](../SGXSupportLevel.md).
+ 
+- A version of Windows OS with native support for SGX features:
+   - For server: Windows Server 2016
+   - For client: Windows 10 64-bit version 1709 or newer
+   - To check your Windows version, run `winver` on the command line.
+
+## Software prerequisites
+- [Microsoft Visual Studio Build Tools 2017](https://aka.ms/vs/15/release/vs_buildtools.exe)
+- [Git for Windows 64-bit](https://git-scm.com/download/win)
+- [OCaml for Windows 64-bit](https://www.ocamlpro.com/pub/ocpwin/ocpwin-builds/ocpwin64/20160113/)
+- [Clang/LLVM for Windows 64-bit](http://releases.llvm.org/7.0.1/LLVM-7.0.1-win64.exe)
+
+## Prerequisites specific to SGX support on your system
+- For systems with support for SGX1  - [Intel's PSW 2.2](WindowsManualSGX1Prereqs.md)
+- For systems with support for SGX1 + FLC - 
+ [Intel's PSW 2.4, Intel's Data Center Attestation primitives and related dependencies](WindowsManualSGX1FLCDCAPPrereqs.md)
+
+## Microsoft Visual Studio Build Tools 2017
+
+Install [Visual Studio Build Tools 2017](https://aka.ms/vs/15/release/vs_buildtools.exe). Choose the "Visual C++ build tools" workload.
+Visual Studio Build Tools 2017's CMake support (ver 3.12 or above) is required for building the Open Enclave SDK.
+Note that cmake in Visual Studio 2019 is not fully supported yet.
+For more information about cmake support, refer to
```

Comment:
Over here, I just reused existing documentation(See https://github.com/openenclave/openenclave/blob/master/docs/GettingStartedDocs/GettingStarted.Windows.md#microsoft-visual-studio-2017)
 @CodeMonkeyLeet I tried out VS 2019 and it appears to be fine. I have updated the docs to say install VS 2017 or VS 2019. This is because the install prereqs script supports 2017 as of now. I have added issue #2178 so we can use 2019 in pre install scripts and remove 2017 from getting started docs.